master/buildbot/status/web/hooks/github.py

```diff
@@ -111,6 +111,43 @@ def handle_push(self, payload):
 
         return changes, 'git'
 
+    def handle_pull_request(self, payload):
+        changes = []
+        number = payload['number']
+        refname = 'refs/pull/%d/head' % (number)
```

Comment:
`(number)` is no different from `number`.  If you'd like to make sure it's a tuple use `(number,)`